### PR TITLE
Add graphtyper recipe

### PR DESCRIPTION
**Pull request by hannespetur**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by hannespetur** [comment]
Please add the "please review & merge" label. @bioconda/core

File: recipes/graphtyper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{% set version = "2.0" %}
+
+package:
+  name: graphtyper
+  version: '{{ version }}'
+
+source:
+  - url: https://github.com/DecodeGenetics/graphtyper/releases/download/v{{ version }}/graphtyper  # [linux]
```
Comment (dpryan79) [inline recipes/graphtyper/meta.yaml]: Can we just compile it? We can use a `git_url` and `git_rev` so we pull in the submodules.

Comment (hannespetur) [reply recipes/graphtyper/meta.yaml]: > Can we just compile it? We can use a `git_url` and `git_rev` so we pull in the submodules.

Sure, I can give it a shot.

**Comment by raivivek** [comment]
@hannespetur I think you should still pull a version of the repository (identified by commit has). An example is [here](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/hail/meta.yaml).

**Comment by hannespetur** [comment]
Ah, I didn't realize that was possible. Thank you @raivivek .

**Comment by hannespetur** [comment]
Alright, I had some problems where cmake wasn't including $PREFIX/include but I have found a workaround. Please re-review.

Best,
Hannes

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**